changelog.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+## grantami-bomanalytics 2.0.0, DATE TBD
+
+### New features
```

Comment:
I think an important part of this release is the breaking change from legislation names to ids

Reply:
Good point. I'll add a new 'backwards-incompatible changes' section, and move the with_stk_records change into it.